src/python-common/ceph/deployment/translate.py

```diff
@@ -184,6 +185,11 @@ def run(self):
                         cmd += " --crush-device-class {}".format(d)
                     cmds.append(cmd)
 
+                elif self.spec.objectstore == 'seastore':
```

Comment:
this block here implies we won't need to ever add things like `--db-devices` or `--wal-devices` when using seastore (they're only in the bluestore block above). Is that correct?

Reply:
Yes.